lib/gui/app.js

```diff
@@ -102,77 +102,75 @@ app.run(() => {
     version: currentVersion
   })
 
-  settings.load().then(() => {
```

Comment:
Just out of curiosity, would moving the `analytics.logEvent('Application start'` line below the `settings.load().then(` line also have fixed the problem? Or does it go deeper than that?

Reply:
Yeah, that would have fixed the issue. This PR changes the whole Angular.js bootstrapping method so we can be sure everything will be ready before we execute anything, protecting us from forgetting to move analytic events call below a certain other instruction, etc.